test/models/tf_ov_model_zoo/gen_plot.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# ==============================================================================
+#  Copyright 2018-2020 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+import argparse
+import errno
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--csv',
+        type=str,
+        help="Input CSV file for genearting chart.\n",
+        action="store")
+    parser.add_argument(
+        '--title',
+        type=str,
+        help="Optional title for chart.\n",
+        default="Chart",
+        action="store")
+    parser.add_argument(
+        '--ylabel',
+        type=str,
+        help="Optional ylabel chart.\n",
+        default="",
+        action="store")
+
+    arguments = parser.parse_args()
+
+    # Check for mandetary parameters
+    if not arguments.csv:
+        raise Exception("Need to specify --csv")
+
+    csv_filepath = os.path.abspath(arguments.csv)
+    if (os.path.isfile(csv_filepath) == False):
+        raise Exception("CSV file not found: " + csv_filepath)
+    benchmark_data = pd.read_csv(csv_filepath)
+
+    plt.figure(figsize=(16, 6))
+    plt.gcf().subplots_adjust(bottom=0.2)
+    plt.grid(color='#95a5a6', linestyle='--', linewidth=1, axis='y', alpha=0.3)
+    plt.title(arguments.title)
+    plt.ylabel(arguments.ylabel)
+    width = 0.2
+    xdata = benchmark_data['Model']
+    xaxis = np.arange(len(xdata))
+    bar_tf = plt.bar(
+        xaxis,
+        benchmark_data['Stock-TF'],
+        width=width,
+        color='lightgray',
+        label="Stock-TF")
+    bar_stockov = plt.bar(
+        x=xaxis + width,
+        height=benchmark_data['Stock-OV'],
```

Comment:
Minor naming nit, but can we have Stock-TF, OV, TF-OV across the board? There's only one OV so we don't need the "Stock-" qualifier for it, and TFOV should be TF-OV just to be consistent.

Reply:
Ok.